migration/config.go

```diff
@@ -0,0 +1,56 @@
+package migration
+
+import (
+	"fmt"
+	"strings"
+)
+
+type DBConfig struct {
+	// address is a combination of host and port.
+	//
+	//  Example: host[:port]
+	address string
+
+	// userCreds is a combination of username and password (if exists).
+	//
+	//  Example: username[:password]
+	userCreds string
+
+	dbName string
+}
+
+func NewDBConfig(host, database, username string) DBConfig {
+	cfg := DBConfig{}
+
+	// Combine ip address and default Postgres port
+	cfg.address = fmt.Sprintf("%s:%s", host, defaultPostgresPort)
```

Comment:
не комбайн їх, для чого ? Кожного разу на сеті ти мусиш розібрати його і потім знов зібрати... лишня робота. 
збережи окремо хост і порт, сетни по дефолту порт і якшо треба буде додати то пересетни його 




```suggestion
        cfg.port = 5432
        cfg.host = host  
```